runtime/src/message_processor.rs

```diff
@@ -104,6 +106,14 @@ fn verify_instruction(
         return Err(InstructionError::RentEpochModified);
     }
 
+    // If account is converted to executable in this instruction, it must need to be rent exempt
+    if pre.executable != post.executable
+        && !pre.executable
+        && !rent_calculator.is_exempt(post.lamports, post.data.len())
```

Comment:
yeah, I get it, but this is too rigid.  the behavior we want is to optionally enforce.  also this doesn't really help address the stake and vote programs' needs